js/level2.js

```diff
@@ -1,93 +1,143 @@
-//Level2
+// Level2
 
 /*
-  Now, when we start level2, it is time to interact with html file. HTML files is a structure
-  of our web-page - a skeleton. It combines all files from the project together, that is how
-  it know what to do. If you check on our index.html file on the bottom you will see tag
-  <script src="js/level1js"></script> - that is where we joined our first js file. But now
-  change level1.js file on our current one - level2.js.
-  Now you are ready to start!
+    Introduction
+    ============
+
+    Welcome to level 2! It is time to interact with HTML.
+
+    Hyper Text Markup Language (HTML) files are the backbone of the internet.
+    Every website you visit is loaded first as an HTML file.
+
+    We won't dwell today on the syntax or history of HTML as a web standard.
```

Comment:
You're right.  None of that is important anyway.  How about this?

> We won't talk too much about HTML today. What you need to know is that HTML
> files function as a sort of skeleton for your page. Our index.html file 
> combines all the files from this project together so that you can open them 
> in your web browser.